### DISCUSSION TITLE
vercel: ERR_INVALID_URL

### DISCUSSION THREAD

**Discussion by estarossa0** [nextauthjs/next-auth#3199] [Help] [answered]
### Failed to deploy on vercel
![image](https://user-images.githubusercontent.com/57002508/141747091-8895923a-0139-4dc0-8b7e-12a05d384a05.png)
I don't know if this is because of how vercel provide its URL or how next-auth parse, but I can't get to deploy on it. I tried to search for any previous issues but I couldn't find any, so maybe this related to v4 cause I'm using `v4.0.0-beta.6`.

Running `next dev` and `next build` on local works fine.
PS: I will be running more tests so I will post an update later

**Comment by balazsorban44** [comment, marked as answer]
Going to be fixed when this gets released https://github.com/nextauthjs/next-auth/pull/3077. No ETA, probably in a few days

**Reply by estarossa0** [reply]
okay thank you 👍

**Reply by muhaimincs** [reply]
This is still happening on v5. Has this been resolve?
